### PR TITLE
Changes to About project page, Disclaimer page, Search page, and other code improvements

### PR DESCRIPTION
**Pull request by seeta-ramaraju**
Code changes related to [G2P-330](https://www.ebi.ac.uk/panda/jira/browse/G2P-330), [G2P-339](https://www.ebi.ac.uk/panda/jira/browse/G2P-339) and other code improvements

### CODE REVIEW

File: src/views/DisclaimerView.vue [hunk 1]
```diff
@@ -2,30 +2,51 @@
...
+      The data from G2P is freely available and shared in good faith. Interested
+      parties can get in touch via
+      <a
+        href="mailto:g2p-help@ebi.ac.uk"
+        style="text-decoration: none"
+        target="_blank"
+        >g2p-help@ebi.ac.uk</a
+      >
+      for further information. Please acknowledge gene2phenotype (<a
         href="http://www.ebi.ac.uk/gene2phenotype/"
```
Comment (dglemos) [inline src/views/DisclaimerView.vue:45]: It should use https

File: src/views/AboutProjectView.vue [hunk 2]
```diff
@@ -13,7 +13,7 @@ export default {
 </script>
 <template>
   <div class="container px-5 py-3" style="min-height: 60vh">
-    <h2 class="pb-3">The Project</h2>
+    <h2 class="pb-3">The G2P Project</h2>
```
Comment (dglemos) [inline src/views/AboutProjectView.vue:16]: The dropdown should also be updated to `The G2P Project`

**Review by dglemos — COMMENTED** [review]
The updates look good!
I only have a couple of comments.

File: src/views/DisclaimerView.vue (hunk 1 above)
Comment (seeta-ramaraju) [reply src/views/DisclaimerView.vue]: changed in latest commit

File: src/views/AboutProjectView.vue (hunk 2 above)
Comment (seeta-ramaraju) [reply src/views/AboutProjectView.vue:16]: changed in latest commit

**Pull request merged**